script/deploy/zksync/DeployRelayFacet.s_1.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.17;
+
+import { DeployScriptBase } from "./utils/DeployScriptBase.sol";
+import { stdJson } from "forge-std/Script.sol";
+import { RelayFacet } from "lifi/Facets/RelayFacet.sol";
+
+contract DeployScript is DeployScriptBase {
+    using stdJson for string;
+
+    constructor() DeployScriptBase("RelayFacet") {}
+
+    function run()
+        public
+        returns (RelayFacet deployed, bytes memory constructorArgs)
+    {
+        constructorArgs = getConstructorArgs();
+
+        deployed = RelayFacet(deploy(type(RelayFacet).creationCode));
+    }
+
+    function getConstructorArgs() internal override returns (bytes memory) {
+        string memory path = string.concat(root, "/config/relay.json");
+        string memory json = vm.readFile(path);
+
+        address relayReceiver = json.readAddress(
+            string.concat(".", network, ".relayReceiver")
+        );
+
+        address relaySolver = json.readAddress(
+            string.concat(".", network, ".relaySolver")
+        );
```

Comment:
I think there should be:
`
        address relayReceiver = _getConfigContractAddress(
            path,
            string.concat(".", network, ".relayReceiver")
        );
        address relaySolver = _getConfigContractAddress(
            path,
            string.concat(".", network, ".relaySolver")
        );
 `